sdk/dynatrace/Azure.ResourceManager.Dynatrace/src/Generated/Models/LinkableEnvironmentListResponse.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Dynatrace.Models
+{
+    /// <summary> Response for getting all the linkable environments. </summary>
+    internal partial class LinkableEnvironmentListResponse
```

Comment:
`XXXResponse` could be a bad name. Previously in `cosmosdb`, I have to rename a lot `XXXGetResult`. See: https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/autorest.md

Reply:
This is an internal class, so I think we don't need to change it.